client/src/utils/attributeAndParamUtils.js

```diff
@@ -501,6 +501,25 @@ const initSsotSessionStorage = (robotId) => {
         console.error(error);
       });
 };
+
+/**
+ * @description Will send a backend call to delete a robot
+ * @param {String} robotId Id of the robot that is deleted
+ */
+const deleteRobotFromDB = async (robotId)  => {
```

Comment:
Although there are already some fetches in this file, I would advise against it, since we have the api folder to route all callouts through methods there. 
Please at least do it with this file, I do not expect you to do it for all methods in this file (but if you really want to, go ahead ;) )